composer_multi_project_monitoring/main.tf

```diff
@@ -0,0 +1,1018 @@
+# [START composer_multi_project_monitoring]
+#   Monitoring for multiple Cloud Composer environments
+#   
+#   Usage:
+#       1. Create a new project that you will use for monitoring of Cloud Composer environments in other projects
+#       2. Store the name of this project in "monitoring_project" variable below, in the locals block
+#       3. Save the list of projects with Cloud Composer environments to be monitored in "monitored_projects", set in the locals block
+#       4. Run "terraform apply"
+#
+#   The script creates the following resources in the monitoring project:
+#           1. Adds monitored projects to Cloud Monitoring
+#           2. Creates Alert Policies
+#           3. Creates Monitoring Dashboard
+#
+
+
+#######################################################
+#  
+# Overall settings
+#
+########################################################
+
+
+locals {
+    monitoring_project = "YOUR_MONITORING_PROJECT"
+    monitored_projects = toset(["YOUR_PROJECT_TO_MONITOR_1", "YOUR_PROJECT_TO_MONITOR_2", "YOUR_PROJECT_TO_MONITOR_3"])
+}
```

Comment:
Nit: per samples style guide using locals is not recommended.

Reply:
Can you point me to the samples style guide and suggest an alternative? I didn't see a style guide linked in [this guide](https://cloud.google.com/guides/authoring/terraform/prepare-terraform) or the related pages

Reply:
This part calls out [only resource blocks](https://cloud.google.com/guides/authoring/terraform/terraform-docs-samples-intro#style) although locals is not explicitly forbidden. I had a doc comment thread with @betsy-lichtenberg who suggested we do expand to locals. In this case since both locals are only used once (not including provider initialization), we can just hard code it where used.

Reply:
SGTM, removed and updated the instruction comments accordingly